src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.ts

```
@@ -293,24 +308,67 @@
+          },
+          secondaryCta: {
+            text: 'Cancel',
+            action: 'CANCEL',
+          },
+        },
+        cssClass: 'pop-up-in-center',
+        backdropDismiss: false,
       });
+    }
+
+  async getCurrentLocation() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)